libraries/shared-gui/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+set(TARGET_NAME shared-gui)
+
+include_directories("${QT_DIR}/include/QtCore/${QT_VERSION}/QtCore" "${QT_DIR}/include/QtCore/${QT_VERSION}")
+
+# TODO: there isn't really a good reason to have Script linked here - let's get what is requiring it out (RegisteredMetaTypes.cpp)
```

Comment:
Is this TODO still valid?

Reply:
I believe so, both `shared-gui` and `shared` still depend on QScript.